lib/management/user.ts

```diff
@@ -521,15 +522,23 @@ const withUser = (sdk: CoreSdk, managementKey?: string) => {
      * Note: The 'Manage tokens from provider' setting must be enabled.
      * @param loginId the login ID of the user
      * @param provider the provider name (google, facebook, etc.).
+     * @param withRefreshToken optional, include the refresh token in the response
+     * @param forceRefresh optional, force to refresh the token
```

Comment:
Need to update this to reflect your change

Reply:
updated